.github/workflows/update-db.yaml

```diff
@@ -0,0 +1,41 @@
+name: Update databases schema
+
+on:
+  push:
+    # Run workflow when the schema changes.
+    branches: [main]
+    paths:
+      - "data-serving/data-service/schemas/**"
+
+jobs:
+  update-case-data:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: Update dev schema
```

Comment:
Believe this is done by `.github/workflows/case-data-update-dev.yml`.

Reply:
I was editing this as you typed sorry ;) I noticed too late.

Reply:
Ah, you've already removed!